src/ucp/wireup/wireup.h

```diff
@@ -224,6 +226,8 @@ double ucp_wireup_iface_bw_distance(const ucp_worker_iface_t *wiface);
 int ucp_wireup_is_lane_connected(ucp_ep_h ep, ucp_lane_index_t lane,
                                  const ucp_address_entry_t *addr_entry);
 
+uct_ep_h ucp_wireup_init_slow_lane(ucp_ep_h ep, ucp_lane_index_t slow_lane_idx);
+
```

Comment:
why needed?

Reply:
bcz it's called from `.inl`, it does not make sense to have slow path as `static inline`